apps/els_lsp/src/els_code_lens.erl

```diff
@@ -8,12 +8,13 @@
 %% Callback Functions
 %%==============================================================================
 
--callback command(poi()) -> els_command:command_id().
--callback command_args(els_dt_document:item(), poi()) -> [any()].
+-callback init(els_dt_document:item()) -> state().
+-callback command(els_dt_document:item(), poi(), state()) ->
```

Comment:
There was very little gain in having two callback functions for a very couple functionality. Use a combined one, instead.